templates/docs/patterns/navigation/index.md

```diff
@@ -43,6 +43,15 @@ By default, dropdowns are left-aligned with their parent; if you'd prefer the me
 View example of the dropdown pattern
 </a></div>
 
+## Sliding
+
+You can add nested dropdowns to the navigation by nesting one or more `p-navigation__dropdown` in `p-navigation__item--dropdown-toggle` components.
+They will be rendered as sliding panels that slide in from the right side of the screen on small screens and as dropdowns on larger screens.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/navigation/sliding-search" class="js-example">
```

Comment:
We don't usually do this, but it would be good to force the narrow view of the example by putting max-width on it.
And there is a setting to make the height of the iframe bigger as well, so dropdowns are visible when opened.

Something like that should work:


```suggestion
<div class="embedded-example" style="max-width: 40rem"><a href="/docs/examples/patterns/navigation/sliding-search" class="js-example" data-height="600">
```